app/decorators/claim_decorator.rb

```diff
@@ -8,4 +8,12 @@ def item_status_tag(association)
       { text: I18n.t("decorators.claim.not_started"), colour: "grey" }
     end
   end
+
+  def status_tag
+    if claim.provider.present? && claim.mentors.any?
+      { text: I18n.t("decorators.claim.submitted"), colour: "blue" }
+    else
+      { text: I18n.t("decorators.claim.not_started"), colour: "grey" }
+    end
+  end
```

Comment:
I believe we have a component for this, `Claim::StatusTagComponent`.

Reply:
https://github.com/DFE-Digital/itt-mentor-services/pull/292/commits/13938894ef6e4b74c38e3a95bbec2c83f18e95ff